src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -5,26 +5,70 @@ import { CountrySelect } from "Styleguide/Components"
 import { Flex, Join, Spacer } from "Styleguide/Elements"
 import { TwoColumnSplit } from "./TwoColumnLayout"
 
-interface AddressFormProps {
-  country?: string
-  onUpdateName?: (e: any) => void
-  onUpdateCountry?: (e: any) => void
-  onUpdatePostalCode?: (e: any) => void
-  onUpdateAddressLine1?: (e: any) => void
-  onUpdateAddressLine2?: (e: any) => void
-  onUpdateCity?: (e: any) => void
-  onUpdateRegion?: (e: any) => void
+export interface Address {
+  name: string
+  country: string
+  postalCode: string
+  addressLine1: string
+  addressLine2: string
+  city: string
+  region: string
+  phoneNumber: string
 }
 
-export class AddressForm extends React.Component<AddressFormProps> {
+export const emptyAddress: Address = Object.freeze({
```

Comment:
Are we polyfilling Object.freeze? I remember seeing Core in our bundle payloads, so I think we probably are, but it'd be good to double check.

Reply:
🤔 do we need to? 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze#Browser_compatibility

Reply:
You make a valid point 😄